arduino-ide-extension/src/browser/settings.tsx

```diff
@@ -13,6 +13,8 @@ import { FileService } from '@theia/filesystem/lib/browser/file-service';
 import { ThemeService } from '@theia/core/lib/browser/theming';
 import { MaybePromise } from '@theia/core/lib/common/types';
 import { WindowService } from '@theia/core/lib/browser/window/window-service';
+import { CommandRegistry } from "@theia/core";
```

Comment:
Please use single quotes. I have to add linter for it. Thanks!

Reply:
nice, we don't use prettier or editorconfig, correct? do we plan to use it in the future?